js/field.js

```diff
@@ -0,0 +1,228 @@
+function setOnClicksAndOtherThings() {
+    
+}
+
+var field = document.getElementById('field');
+var parentCard;
+var tempCard;
+var totalCountInAcePiles=0;
+field.onmousedown = onMouseDown;
+
+function onMouseDown(e) {
+    function isDragable(element) {
+        if (element.classList.contains('dragable')) {
```

Comment:
I don't think that it is the right way to use style class here. You'd better use data-attribute.